src/core/lib/backoff/backoff.cc

```diff
@@ -20,23 +20,23 @@
 
 #include <grpc/support/useful.h>
 
-void grpc_backoff_init(grpc_backoff *backoff,
-                       grpc_millis initial_connect_timeout, double multiplier,
-                       double jitter, grpc_millis min_timeout_millis,
-                       grpc_millis max_timeout_millis) {
-  backoff->initial_connect_timeout = initial_connect_timeout;
+void grpc_backoff_init(grpc_backoff *backoff, grpc_millis initial_backoff,
+                       double multiplier, double jitter,
+                       grpc_millis min_backoff, grpc_millis max_backoff) {
+  backoff->initial_backoff = initial_backoff;
   backoff->multiplier = multiplier;
   backoff->jitter = jitter;
-  backoff->min_timeout_millis = min_timeout_millis;
-  backoff->max_timeout_millis = max_timeout_millis;
+  backoff->min_backoff = min_backoff;
+  backoff->max_backoff = max_backoff;
   backoff->rng_state = (uint32_t)gpr_now(GPR_CLOCK_REALTIME).tv_nsec;
 }
 
 grpc_millis grpc_backoff_begin(grpc_exec_ctx *exec_ctx, grpc_backoff *backoff) {
-  backoff->current_timeout_millis = backoff->initial_connect_timeout;
-  const grpc_millis first_timeout =
-      GPR_MAX(backoff->current_timeout_millis, backoff->min_timeout_millis);
-  return grpc_exec_ctx_now(exec_ctx) + first_timeout;
+  backoff->current_backoff = backoff->initial_backoff;
+  const grpc_millis current_deadline =
+      grpc_exec_ctx_now(exec_ctx) + backoff->initial_backoff;
+  return GPR_MAX(current_deadline,
+                 grpc_exec_ctx_now(exec_ctx) + backoff->min_backoff);
```

Comment:
If we intend to make the returne value to be `current_deadline` here, this should not be max'ed with min_backoff (min_timeout is a more appropriate name I think). Max'ing should be taken care of by caller when they make TryConnect.